Forge/src/main/java/com/blamejared/crafttweaker/platform/ForgePlatformHelper.java

```diff
@@ -197,7 +213,7 @@ public Method findMethod(Class<?> type, String methodName, final Class<?> return
         try {
             return ObfuscationReflectionHelper.findField(castToSuperExplicitly(clazz), fieldName);
         } catch(final ObfuscationReflectionHelper.UnableToFindFieldException e) {
-            throw new HandleHelper.UnableToLinkHandleException("Field %s was not found inside class %s".formatted(StringUtils.quoteAndEscape(fieldName), clazz.getName()), e);
+            throw new HandleUtil.UnableToLinkHandleException("Field %s was not found inside class %s".formatted(StringUtil.quoteAndEscape(fieldName), clazz.getName()), e);
```

Comment:
`.getName()` or `.getQualifiedName()` or whatever the name?

Reply:
`getName` is how the fully qualified name is obtained